its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/CodeCoverageTest.java

```diff
@@ -97,6 +97,32 @@ void whenRunningOnAzureDevops_coverageIsImported() throws IOException {
     assertThat(endStepResult.getLogs()).contains("Coverage Report Statistics: 2 files, 1 main files, 1 main files with coverage, 1 test files, 0 project excluded files, 0 other language files.");
   }
 
+  @Test
+  void dotCover_CoverageDirectoryIsNotImported() throws Exception {
+    var projectName = "DotCoverExcludedCoverage";
+    var projectDir = TestUtils.projectDir(basePath, projectName);
+    var token = TestUtils.getNewToken(ORCHESTRATOR);
+
+    ORCHESTRATOR.getServer().provisionProject(projectName, projectName);
+    var scanner = TestUtils.newScanner(ORCHESTRATOR, projectDir, token)
+      .addArgument("begin")
+      .setProjectKey(projectName)
+      .setProjectName(projectName)
+      .setProperty("sonar.projectBaseDir", projectDir.toAbsolutePath().toString())
+      .setProperty("sonar.verbose", "true")
+      .setProperty("sonar.cs.dotcover.reportsPaths", "dotCover.Output.html")
+      .setProjectVersion("1.0");
+    var beginStepResult = ORCHESTRATOR.executeBuild(scanner);
+    assertTrue(beginStepResult.isSuccess());
+
+    TestUtils.runDotnetCommand(projectDir, "build", "--no-incremental");
+    var endStepResult = TestUtils.executeEndStepAndDumpResults(ORCHESTRATOR, projectDir, projectName, token);
+    assertTrue(endStepResult.isSuccess());
+    assertThat(TestUtils.allIssues(ORCHESTRATOR)).hasSize(1)
+      .extracting(Issues.Issue::getRule)
+      .containsOnly("csharpsquid:S2699");
```

Comment:
If you take out my change with the coverage directory, this test fails because it finds a lot of issues in the css/html/js files of dotCover's directory.